recipes/spiceypy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "spiceypy" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "33db27531f7e8e31ce06970231b3265fe043b608d1404c9f43b2af14508713f3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.zip
+  url: https://github.com/AndrewAnnex/SpiceyPy/archive/v{{ version }}.zip
+  sha256: {{ sha256 }}
+
+  patches:
+    - 0001-Remove-CSPICE-installation-logic.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  noarch_python: True
+
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - cspice ==66
+    - numpy
+    - six
+
+test:
+  imports:
+    - "spiceypy"
+    - "spiceypy.utils"
+
+about:
+  home: https://github.com/AndrewAnnex/SpiceyPy
+  license: MIT
+  summary: 'The NASA JPL NAIF SPICE toolkit wrapper written in Python'
+
+  description:
+    - |
+      SpiceyPy is a python wrapper for the SPICE Toolkit. SPICE is an
+      essential tool for scientists and engineers alike in the planetary
+      science field for Solar System Geometry.
+  doc_url: http://spiceypy.readthedocs.io/
+
+extra:
+  recipe-maintainers:
+    - juanlu001
+    - AndrewAnnex
```

Comment:
Are you ok being a maintainer here, @AndrewAnnex?

Reply:
yeah I asked to be included, but I think @Juanlu001 will be the lead maintainer.